### PR TITLE
bump CNAO to 0.23.0

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

**Comment by phoracek** [comment]
@tiraboschi @rthallisey could you take a look please?

File: hack/defaults [hunk 1]
```diff
@@ -4,7 +4,7 @@ TEMP_DIR=`mktemp -d`
 WAIT_TIMEOUT="${WAIT_TIMEOUT:-300s}"
 
 # Use upstream manifests with downstream containers
-KUBEVIRT_MANIFEST_VERSION="v0.17.0"
+KUBEVIRT_MANIFEST_VERSION="v0.23.0"
```
Comment (tiraboschi) [inline hack/defaults]: this is wrong, you have to edit NETWORK_ADDONS_MANIFEST_VERSION two lines below

Comment (phoracek) [reply hack/defaults]: Fixed

**Comment by rthallisey** [comment]
Might be an oom flake.
```
ovs-cni-amd64-4x66h                                   1/2     CrashLoopBackOff   6          7m58s
ovs-cni-amd64-7fv86                                   1/2     CrashLoopBackOff   6          7m58s
```

**Comment by rthallisey** [comment]
ci test please

**Comment by phoracek** [comment]
@rthallisey it is not OOM in this case, we have an issue on Kubernetes cluster (without OVS installed). I'm working on the resolution in https://github.com/kubevirt/ovs-cni/pull/94

**Pull request closed without merging**